testsuite/tests/typing-layouts-products/basics_alpha.ml

```diff
@@ -66,7 +66,27 @@ type t2 : any_non_null
 Line 3, characters 0-39:
 3 | type t3 : any mod non_null = #(t1 * t2);;
     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Error: The kind of type "#(t1 * t2)" is any & any
+Error: The kind of type "#(t1 * t2)" is
+         any mod global with t1
+t2 unique with t1
+t2 many with t1
+t2
+                 uncontended with t1
+t2 portable with t1
+t2
+                 external_ with t1
+t2 non_null with t1
+t2
+         & any mod global with t1
+t2 unique with t1
+t2 many with t1
+t2
+                   uncontended with t1
+t2 portable with t1
+t2
+                   external_ with t1
+t2 non_null with t1
+t2
```

Comment:
This formatting is pretty terrible. But I think it's okay to leave this for now because:
a) I don't think any users outside our team will see an error like this
b) We plan to fix printing soon

Reply:
> b) We plan to fix printing soon
 
extremely soon:

https://github.com/ocaml-flambda/flambda-backend/pull/3410